boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -20,7 +20,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,ccm = &ccm0;
-		zephyr,display = &ili9341;
+		zephyr,display = &ltdc;
```

Comment:
This modification is not sufficient.
Node should also be enabled, maybe pins configured, ...